src/navigation/RootNavigator.tsx

```diff
@@ -111,6 +112,10 @@ const RootNavigator = () => {
     dispatch(appSlice.actions.toggleConnectedWallets())
   }, [dispatch])
 
+  useEffect(() => {
+    dispatch(walletRestApi.util.resetApiState())
```

Comment:
Will this fix the price not updating? I noticed that happening again on my build.

Reply:
Yup. What is happening is that RTK query cache is staying on peoples redux state. I couldn't figure out a way to check if the state was empty or not. Open to suggestions! 🙂